src/sentry/static/sentry/app/components/previewFeature.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import Alert from 'app/components/alert';
+
+type Props = {
+  type?: 'success' | 'error' | 'warning' | 'info';
```

Comment:
This can reference `Alert['type']`

Reply:
cool, wasn't sure if I should do that considering that alert has two more types (muted, beta) and didn’t know if it makes sense with previewFeature component… thanks, changed it 👍